R/Utilities.R

```diff
@@ -1022,7 +1022,7 @@ getFeatureLayer <- function(url, bbox = NULL) {
 #' Download a shapefile from an API and save it to a local folder, overwriting existing file if it exists
 #' writeLayer is used by TADA_UpdateTribalLayers in TADAGeospatialRefLayers.R.
 #'
-#' @param urlURL of the layer REST service, ending with "/query". Example: https://geopub.epa.gov/arcgis/rest/services/EMEF/Tribal/MapServer/2/query (American Indian Reservations)
```

Comment:
@renaemyers FYI this is the only issue I ran into (missing a space here) related to your additions. I fixed this in the last PR.